src/accelerate/accelerator.py

```diff
@@ -926,6 +926,7 @@ def prepare_model(self, model: torch.nn.Module, device_placement=None):
             kwargs = self.ddp_handler.to_kwargs() if self.ddp_handler is not None else {}
             model = torch.nn.parallel.DistributedDataParallel(model, **kwargs)
         if self.native_amp:
+            model._original_forward = model.forward
```

Comment:
@sgugger I know how you feel about `_` but I felt this was a decent time for one because the user really *shouldn't* need to know this exists at all or try to accidentally call it in any way

Reply:
Agree in this case :-)